resources/lib/jellyfin/core/configuration.py

```diff
@@ -38,14 +40,16 @@ def __shortcuts__(self, key):
         return
 
     def __getitem__(self, key):
+        LOG.debug("__getitem__(%r)", key)
         return self.data.get(key, self.__shortcuts__(key))
 
     def __setitem__(self, key, value):
         self.data[key] = value
 
     def app(self, name, version, device_name, device_id, capabilities=None, device_pixel_ratio=None):
-        
         LOG.info("Begin app constructor.")
+        # import traceback
+        # LOG.debug(''.join(['\n'] + traceback.format_stack()))
```

Comment:
Remove or uncomment

Reply:
Right, that's from attempting to trace down what's happening with the `config['app.name']` being `None` exception.
I didn't want to lose it, so I commented it out in stead of removing. Fixed now.